composer.json

```diff
@@ -35,13 +35,13 @@
     "suggest": {
         "sonata-project/admin-bundle": "To be able to use MessageAdmin.",
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
-        "guzzle/guzzle" : "^3.9",
+        "guzzlehttp/guzzle" : "^3.9",
```

Comment:
These are the only lines with Guzzle usage:
https://github.com/sonata-project/SonataNotificationBundle/blob/85bd86a34e6139f1b92d45be6769c539eb8e7366/Backend/AMQPBackendDispatcher.php#L200-L209